lib/rubocop/cop/rspec/factory_bot/attribute_defined_statically.rb

```diff
@@ -111,25 +109,31 @@ def method_uses_parens?(location)
             location.begin.source == '(' && location.end.source == ')'
           end
 
-          def autocorrect_hash_without_braces(node)
-            autocorrect_replacing_parens(node, ' { { ', ' } }')
-          end
+          def autocorrect_replacing_parens(node)
+            left_braces, right_braces = braces(node)
 
-          def autocorrect_replacing_parens(node,
-                                           start_token = ' { ',
-                                           end_token = ' }')
             lambda do |corrector|
-              corrector.replace(node.location.begin, start_token)
-              corrector.replace(node.location.end, end_token)
+              corrector.replace(node.location.begin, ' ' + left_braces)
```

Comment:
What makes the `' ' +` padding necessary?

Reply:
Without it `attribute(value)` would turn into `attribute{ value }` instead of `attribute { value }` We don't need the extra space when there are no parens because there is already a space  
(`attribute value` becomes `attribute { value }`.